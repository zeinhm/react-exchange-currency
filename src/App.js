import React from 'react';
import pages from './pages'
import AppContextProvider from './contexts'

function App() {
  return (
    <AppContextProvider>
      <pages.Home />
    </AppContextProvider>
  );
}

export default App;
