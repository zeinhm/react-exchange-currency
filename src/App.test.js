import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow';
import App from './App';

describe('src/App', () => {
  test('render', () => {
    const shallow = new ShallowRenderer();
    const tree = shallow.render(<App />);
    expect(tree).toMatchSnapshot();
  });
});
