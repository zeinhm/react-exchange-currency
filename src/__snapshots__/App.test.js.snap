// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/App render 1`] = `
<AppContextProvider>
  <suspense />
</AppContextProvider>
`;
